vendor/src/github.com/microsoft/hcsshim/createprocess.go

```diff
@@ -69,11 +112,26 @@ func CreateProcessInComputeSystem(id string, params CreateProcessParams) (proces
 
 	logrus.Debugf(title+" - Calling the procedure itself %s %s", id, paramsJson)
 
+	var stdinHandle, stdoutHandle, stderrHandle syscall.Handle
+	var stdinParam, stdoutParam, stderrParam uintptr
+	if useStdin {
+		stdinParam = uintptr(unsafe.Pointer(&stdinHandle))
```

Comment:
@jhowardmsft @jstarks `stdinHandle` is a `syscall.Handle`, which is already a `uintptr` shouldn't we drop the `&`?